config/routes.rb

```diff
@@ -37,7 +37,7 @@
       end
 
       scope path: ":job_id" do
-        resource :job_application, only: %i[new create]
+        resource :job_application, only: %i[new create show]
```

Comment:
I think this should be a `show` on line 30 so we access job application by job application rather than the job id